pin/page_objects/base_page_object.py

```diff
@@ -0,0 +1,11 @@
+from selenium_tests.base_page_object import BasePageObject
```

Comment:
the base /pin directory (and all its files) looks like it can be removed entirely, since you have pin logic in /selenium_tests/page_objects/pin_page.py.